### DISCUSSION TITLE
Any privacy issues while using copilot for office projects?

### DISCUSSION THREAD

**Discussion by rthangam** [community/community#7819] [Copilot Conversations] [answered]
Since copilot suggests based on our code want to know if it is secure enough to use for office projects ?

**Comment by MulleDK19** [comment, marked as answer]
What do you mean?

Code snippets, accepted suggestions and edited suggestions are sent to the Copilot team. Read more about their telemetry policy here: https://docs.github.com/en/github/copilot/about-github-copilot-telemetry
